contracts/api-spec.yaml

```diff
@@ -365,6 +376,29 @@ paths:
 
 components:
   schemas:
+    LogsModel:
+      description: A list of all logs
+      type: array
+      items:
+        $ref: '#/components/schemas/LogModel'
+
+    LogModel:
+      description: a log object
+      properties:
+        id:
+          type: integer
+          format: int64
+          example: '123'
+        dateTime:
+          type: string
+          format: date-time
+        message:
+          type: string
+          example: 'A log message'
+        level:
+          type: string
+          example: 'info'
```

Comment:
We also need to know what device created the log. This will be useful so we can view device-specific logs. That requirement isn't captured in this implementation IMO but i'm willing to merge this and add it later.

Reply:
That i put it in the log message.

Reply:
e.g. Device with serial number 4 has been created. 4 is orginally a var

Reply:
yeah, but we'll need to represent it in the database entry too so we can query for a device's logs. agreed, we'll take a look again next sprint